src/config.rs

```diff
@@ -429,14 +465,18 @@ impl RecertConfig {
                 },
                 cluster_rename: cli.cluster_rename,
                 hostname: cli.hostname,
-                ip: cli.ip,
+                ip: if cli.ip.is_empty() { None } else { Some(cli.ip) },
```

Comment:
The `ClusterCustomizations` struct should just hold a `Vec`, no need for `Option` to wrap a `Vec`